src/clib/pio_spmd.c

```diff
@@ -69,6 +71,7 @@ int pair(int np, int p, int k)
  * @param comm MPI communicator for the MPI_Alltoallw call.
  * @param fc pointer to the struct that provided flow control options.
  * @returns 0 for success, error code otherwise.
+ * @author Jim Edwards
```

Comment:
I don't understand why these @author lines are showing up as changes added in your PR. They are already in the code base. This is happening a lot, if we can figure it out we can reduce many of the changes.